xchange-lgo/src/main/java/org/knowm/xchange/lgo/dto/product/LgoProductCurrency.java

```diff
@@ -14,7 +15,7 @@ public LgoProductCurrency(
       @JsonProperty("increment") BigDecimal increment,
       @JsonProperty("limits") LgoLimit limits) {
     this.id = id;
-    this.increment = increment;
+    this.increment = increment != null ? increment.stripTrailingZeros() : null;
```

Comment:
DTOs should represent what came from the exchange, this is premature adaptation and should not be here

Reply:
Fix. Thank you.